ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -37,7 +37,11 @@ import SpyInstance = jest.SpyInstance;
 
 type AnyWrapper = (ShallowWrapper|ReactWrapper);
 
-jest.mock('app/utils/workbench-gapi-client', () => ({
+jest.mock('app/utils/navigation', () => ({
```

Comment:
I can't figure out how to make the import mock and the regular mock work at the same time. Please advise :(